daemon/main.go

```diff
@@ -138,6 +138,11 @@ func init() {
 						Name:        "keep-config",
 						Usage:       "When restoring state, keeps containers' configuration in place",
 					},
+					cli.BoolFlag{
+						Destination: &config.KVStoreIPv4Registration,
```

Comment:
I think we should push the address unconditionally if the KVStore is available. We can think about a mode which does not require a KVStore in which forwarding is performed without policy enforcement.